File: runtimes/dotnet-8.0/src/RuntimeOutput.cs
namespace DotNetRuntime
{
    public class RuntimeOutput
    {
        public byte[] Body { get; set; }
        public int StatusCode { get; set; }
        public Dictionary<string, string> Headers { get; set; }

        public bool Chunked { get; set; }
        public RuntimeOutput(byte[] body, int statusCode, Dictionary<string, string> headers, bool chunked)
        {
            Body = body;
            StatusCode = statusCode;
            Headers = headers;
            Chunked = chunked;
        }
    }
}

File: runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeOutput.java
package io.openruntimes.java;

import java.util.Map;

public class RuntimeOutput {
    private final byte[] body;
    private final int statusCode;
    private final Map<String, String> headers;
    private final boolean chunked;

    public RuntimeOutput(
            byte[] body,
            int statusCode,
            Map<String, String> headers,
            boolean chunked
    ) {
        this.body = body;
        this.statusCode = statusCode;
        this.headers = headers;
        this.chunked = chunked;
    }

    public byte[] getBody() {
        return this.body;
    }

    public int getStatusCode() {
        return this.statusCode;
    }

    public Map<String, String> getHeaders() {
        return this.headers;
    }

    public boolean getChunkedStatus() {
        return this.chunked;
    }

}
